pom.xml

```diff
@@ -48,14 +48,14 @@
     <releaseProfiles> </releaseProfiles>
     <arguments> </arguments>
     <argLine>-Xmx768M -Djava.awt.headless=true</argLine>
-    <jenkins.version>1.625.3</jenkins.version>
+    <jenkins.version>2.60.1</jenkins.version>
```

Comment:
I would suggest using `2.7.3` and reverting `java.level` as a more conservative default. The default for `jenkins.version` should not matter much because most plugins override it anyway, but I am afraid a lot leave `java.level` at the default; so if they inherit 8 but are in fact depending on something older than 2.60, there will be problems (pending https://github.com/jenkinsci/jenkins/pull/3016).

(Plugin archetypes specify both explicitly and I would recommend this always, but AFAIK it is not possible to force them to be overridden.)

Reply:
I looked at the [jenkins stats](http://stats.jenkins.io/jenkins-stats/svg/201709-jenkins.svg), 2.60.1 seemed to be a sweet spot.  those on older cores tend not to upgrade plugins as much anyway.

You can run older cores on newer JDKs and use it everywhere (agents etc..)  (a good proportion of users on < 2.60.1 likely do this), there are some that do not - but for those if they upgrade a plugin that requires a newer JDK and it bombs out they can raise a JIRA for the affected plugin no, or roll back that plugin.

by the same argument  you could also say people set the core to something higher than 2.60 and do not update java.level to 8 to take advantage of new features in the language, yes it does not blow up jenkins but then...

I will go with the consensus - but I lean towards 2.60 and java 8

Reply:
btw - the conservative change to 2.7 and removing the servlet dance is #77

Reply:
Could we meet somewhere in the middle? E.g. 2.32 or 2.46. Just to keep Java7 compatibility by default for a while

Reply:
I’d rather just go with 2.60.1 we have a major version bump, let’s just Java 8 all the things. If we default Java 7 then i’d say keep the parent as 2.x

Reply:
Reading http://stats.jenkins.io/plugin-installation-trend/capabilities.json it would mean that this change means that a plugin targetting this will _only_ hit 48% of the installs out there. (74623/154880).

Now the reality is more complex than that IMO, out of that 154880 installs, many are probably stuck in the past and anyway don't and won't upgrade ever.

But as this all goes with a major version bump, I'm OK with this I think.